spec/cart_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'cart'
+require 'dish'
+require 'viewer'
+require 'menu'
+
+meat_feast = Dish.new("Meat Feast", 950)
+four_seasons = Dish.new("Four Seasons", 650)
+veggie = Dish.new("Veggie", 700)
+margherita = Dish.new("Margherita", 500)
+
+cart = Cart.new
+
```

Comment:
and you can do this in a `before` block

Reply:
but perhaps I am misunderstanding your intention here, since you use a before block below...